static/js/components/PrivacyForm.js

```diff
@@ -15,10 +15,16 @@ class PrivacyForm extends ProfileFormFields {
   };
 
   render() {
+    const emailOptions = [
+      { value: "true", label: `Faculty and staff can send me emails about the program`},
+      { value: "false", label: `I don't want to receive any emails` }
```

Comment:
is there a reason you're declaring the labels as template literals? I think strings are preferred unless you need to do interpolation.